__device-tests__/gutenberg-editor-sanity-test-4-visual.test.js

```diff
@@ -216,7 +216,9 @@ describe( 'Gutenberg Editor - Test Suite 4', () => {
 
 			// Open and wait for block settings
 			await editorPage.openBlockSettings();
-			await editorPage.driver.sleep( 500 );
+			// On Android, wait extra time before taking the screenshot
+			// to let the bottom sheet's scrollbar fade out.
+			await editorPage.driver.sleep( isAndroid() ? 2000 : 500 );
```

Comment:
Thinking out loud — an alternative is to focus the screenshot on a specific element or crop the screenshot by a few pixels. Genuinely uncertain if that approach is appropriate for the spirit of this test, merely offering it for discussion. WDYT?

Reply:
Not sure if we can focus on a specific element as the test case aims to check how the block and settings look in dark mode. However, I agree that alternatively to increase the waiting time we could crop the screenshot by a few pixels from the right to exclude the scrollbar. I don't think we'll miss any element by cropping it a little.

Reply:
I'll update the changes with the above proposal.

Reply:
@dcalhoun We don't have an easy way to crop the screenshot from the right and bottom edges, so I went ahead and added a new parameter to `takeScreenshot` util (https://github.com/wordpress-mobile/gutenberg-mobile/pull/6135/commits/193d6836afcaa2403e361ba9bf784aca56d87f3e).

Additionally, the screenshot and test case have been updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6135/commits/961ea91a390194abfa59a749f450571dad233e27 and https://github.com/wordpress-mobile/gutenberg-mobile/pull/6135/commits/eb584b3b16ff1b8307165ae67611142e649922f8.

Let me know if you could take another look at the PR changes, thanks 🙇 !